invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/details.html

```diff
@@ -351,3 +351,47 @@ <h3 class="ui header">{{ _('References') }}</h3>
     <div class="ui divider"></div>
   {% endif %}
 {% endif %}
+
+{% if record.custom_fields["legacy:subjects"] %}
```

Comment:
I tested with an all-clean almost empty record, and it doesn't give any errors. But I could add another check just in case. LMK

Reply:
thank you for checking! I just realized, could we move this to zenodo-rdm instead of. adding to invenio-app-rdm? The field assumes we have `legacy:subjects`, but this is too zenodo-specific to be added in the core app - other instances will not probably have this field

Reply:
You're 100% right, I'll move it!